Library/Homebrew/formula.rb

```diff
@@ -1271,7 +1271,7 @@ def to_hash
         "root_url" => bottle_spec.root_url,
       }
       bottle_info["files"] = {}
-      bottle_spec.collector.keys.each do |os|
+      bottle_spec.collector.each_key do |os|
```

Comment:
Are you sure? My Ruby 1.8.7 documentation is claiming such an instance method for `Hash` exists and local testing with version 1.8.7-p374 confirmed that.